pkgs/by-name/ne/netmaker/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, installShellFiles
+, nix-update-script
+, ...
+}:
+buildGoModule rec {
+  pname = "netmaker";
+
+  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch=develop" ]; };
+  version = "unstable-2024-01-24";
+
+  src = fetchFromGitHub {
+    owner = "nazarewk";
```

Comment:
Again, why not official repository ?

Reply:
added note in a description

Reply:
I've noticed, sot after your patches are merged into official repository we will be switching back to official repository?
I don't know what's best practice in nixpkgs about things like this

Reply:
yeah, this is the idea

Reply:
I'm really not sure what's the best practice about this within the nixpkgs, so we will need to discuss with someone more involved.